signup-service/src/main/java/io/mosip/signup/services/RegistrationService.java

```diff
@@ -184,6 +187,10 @@ public VerifyChallengeResponse verifyChallenge(VerifyChallengeRequest verifyChal
 
     public RegisterResponse register(RegisterRequest registerRequest, String transactionId) throws SignUpException {
         RegistrationTransaction transaction = cacheUtilService.getChallengeVerifiedTransaction(transactionId);
+        if(!registerRequest.getUsername().equalsIgnoreCase(registerRequest.getUserInfo().get(userNameHandle).asText())) {
```

Comment:
This logic should be part if the esignet profile registry plugin @balaji-alluru

Reply:
Moved the logic to profile registry plugin, hence closing the pr.